### PR TITLE
Feat/testimonial

### PR DESCRIPTION
**Pull request by SaistaShaikh**

### CODE REVIEW

File: src/assets/images/dot.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
+<ellipse cx="10.1383" cy="10" rx="9.41352" ry="10" fill="#9E7BFE"/>
```
Comment (ulhas) [inline src/assets/images/dot.svg:2]: Indentation is off here.

File: src/components/avatar.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  name: string
+  title: string
+  onClick: () => void
+}
+
+const Avatar: FC<AvatarProps> = ({ onClick, ...restProps }) => {
+  return (
+    <div className="flex gap-[5px] sm:gap-[16px] items-center rounded-full">
+      {restProps.active ? (
+        <img
```
Comment (ulhas) [inline src/components/avatar.tsx:15]: We can separate this into small components and use it here

**Review by ulhas — CHANGES_REQUESTED** [review]

File: src/components/avatar.tsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+          className="object-cover rounded-full h-[45px] w-[45px] md:h-[80px] md:w-[80px] lg:h-[110px] lg:w-[110px] border-2 sm:border-4 border-indigo-600"
+          src={restProps.imageURL}
+          alt=""
+        />
+      ) : (
+        <div
```
Comment (pallavisaraogi) [inline src/components/avatar.tsx:21]: make this as a component and same can be use for active state by passing active classes

File: src/components/testimonial.tsx [hunk 4]
```diff
@@ -1,84 +1,77 @@
...
-            <img
-              className="object-cover h-full w-full rounded-full"
-              src="https://media.istockphoto.com/photos/african-american-graduate-student-studying-from-home-during-covid19-picture-id1300821206?s=612x612"
-              alt=""
-            />
+  const handleActive = (index: number) => {
+    const array = [...testimonialArray]
+    const temp = array[index]
+    array[index] = array[activeIndex]
+    array[activeIndex] = temp
```
Comment (pallavisaraogi) [inline src/components/testimonial.tsx:16]: here activeIndex can be set directly by `setActiveIndex`. why swapping?

File: src/components/testimonial.tsx [hunk 5]
```diff
@@ -1,84 +1,77 @@
...
           </div>
-          <div className="h-[32px] w-[32px] md:h-[60px] md:w-[60px] lg:h-[80px] lg:w-[80px] rounded-full">
-            <img
-              className="object-cover h-full w-full rounded-full"
-              src="https://media.istockphoto.com/photos/african-american-graduate-student-studying-from-home-during-covid19-picture-id1300821206?s=612x612"
-              alt=""
-            />
+        </div>
+
+        <div className=" mx-4 py-4 sm:w-3/4 shadow-xl rounded-2xl relative">
```
Comment (pallavisaraogi) [inline src/components/testimonial.tsx:42]: break this into components for showing active testimonial details

**Pull request closed without merging**